Cargo.toml

```diff
@@ -1,7 +1,8 @@
 [package]
 name = "fabricate"
 version = "0.1.0"
-authors = ["geometrically <jai.a@tuta.io>"]
+#Team members, please add your emails and usernames
+authors = ["geometrically <jai.a@tuta.io>", "Redblueflame <contact@redblueflame.com>", "Adledfyr", "cfanoulis", "jaoh1"]
```

Comment:
Might want to put people who are only part of the team, jaoh1 has only made one pull request for something minor

Reply:
Ok !

Reply:
Also, `"Aeledfyr <aeledfyr@gmail.com>"` for me

Reply:
Fixed in `8be5957`